api/apps/api/src/modules/projects/blm/domain/blm-values-calculator.ts

```diff
@@ -0,0 +1,26 @@
+export class BlmValuesCalculator {
```

Comment:
While the algorithm of calculation could indeed belong do domain (although rather technical one), we would rather need to introduce it as an aggregate (gettable, save-able). As it may not be worth the effort atm, I would stick to keeping it within application.

Reply:
If you mean making it an Aggregate Root, as (correct me if I'm wrong) Project could be, I don't think this calculator could be one.
As far as I know, we can also have domain services which encapsulates domain logic (as I thought this was). Also that's why I didn't  make it a NestJS provider, to keep it clean from libraries as I see it as business logic code.
If we make it a provider, we could mock up the results, for example, but I don't really see an scenario where we wouild need that 🤔

Reply:
@AaronPerezPerez you may be right about aggregate.

Calculator rather sounds as `domain policy` so we can keep it within domain.

However, we should try to avoid `static` thing - as otherwise we cannot "replace" policies in application, as we strictly hardcode usage via concrete class.

Reply:
Okeey, refactoring it!